**Prob. 15** Hypotheses Write the null and alternative hypotheses you would use to test each of the following situations:

a) A governor is concerned about his “negatives”—the percent- age of state residents who express disapproval of his job per- formance. His political committee pays for a series of TV ads, hoping that they can keep the negatives below 30%. They will use follow-up polling to assess the ads’ effectiveness.

b) Is a coin fair?

c) Only about 20% of people who try to quit smoking succeed.
Sellers of a motivational tape claim that listening to the recorded messages can help people quit.


**Answers to Prob. 15** 

(a) Let p = proportion of negatives in a series of TV ads.  Then $H_0:p$ = 0.30 vs. $H_A:p$ =/ 0.30.

(b) Let p = proportion for fliping a coin that is fair.  Then $H_0:p$ = 0.50 vs. $H_A:p$ =/ 0.50.

(c) Let p = proportion of people who try to quit smoking succeed.  Then $H_0:p$ = 0.20 vs. $H_A:p$ =/ 0.20.

**Prob. 16** More hypotheses Write the null and alternative hypotheses you would use to test each situation.

a) In the 1950s, only about 40% of high school graduates went on to college. Has the percentage changed?

b) Twenty percent of cars of a certain model have needed costly transmission work after being driven between 50,000 and 100,000 miles. The manufacturer hopes that a redesign of a transmission component has solved this problem.

c) We field-test a new-flavor soft drink, planning to market it only if we are sure that over 60% of the people like the flavor.


**Answers to Prob. 16**

(a) Let p = proportion of high school graduates went on to college.  Then $H_0:p$ = 0.40 vs. $H_A:p$ =/ 0.40.

(b) Let p = proportion of cars of a certain model have needed costly transmission work after being driven between 50,000 and 100,000 miles.  Then $H_0:p$ = 0.20 vs. $H_A:p$ =/ 0.20.

(c) Let p = proportion of the people like the new-flavor soft drink.  Then $H_0:p$ = 0.60 vs. $H_A:p$ =/ 0.60.

**Prob. 42** TV ads A start-up company is about to market a new computer printer. It decides to gamble by running commercials during the Super Bowl. The company hopes that name recognition will be worth the high cost of the ads. The goal of the company is that over 40% of the public recognize its brand name and associate it with computer equipment. The day after the game, a pollster contacts 420 randomly chosen adults and finds that 181 of them know that this company manufactures printers. Would you rec- ommend that the company continue to advertise during Super Bowls? Explain.

In [16]:
from math import sqrt
from scipy.stats import norm

In [20]:
def p42():
    n = 420
    k = 181
    p_hat = k / n
    SE = sqrt( p_hat * (1-p_hat) / n)
    z = norm.ppf(0.975)  
    ME = z * SE
    print(f"95% CI: ({(p_hat - ME)*100:.1f}%, {(p_hat + ME)*100:.1f}%)")
    
p42()

95% CI: (38.4%, 47.8%)


In [13]:
def p42():
    n = 420
    k = 181
    p_hat = k / n
    p0 = 0.4
    q0 = 1 - p0
    sd_p_hat = sqrt(p0 * q0 / n)
    z = (p_hat - p0) / sd_p_hat
    p_val = 1 * norm.cdf(z)
    
    print (f"p_hat: {p_hat:.3f}")
    print (f"sd_p_hat: {sd_p_hat:.3f}")
    print (f"z: {z:.3f}")
    print(f"p-value: {p_val:.3f}")
p42()

p_hat: 0.431
sd_p_hat: 0.024
z: 1.295
p-value: 0.902


**Ansers to Prob. 42** We are 95% confident that the public recognization rate by running commercials during the Super Bowl is between 38.4% and 47.8%. This interval includes the original estimate of 40%, however, we may still hardly claim that running commercials during the Super Bowl increases the public recognization.

Let p = proportion of the public recognize the company's brand name and associate it with computer equipment.  $H_0:p$ = 0.40 vs. $H_A:p$ =/ 0.40.  Because the p-value of 0.902 is large, I retain the null hypothesis.  These survey data provided strong evidence that the public recognization of the company and its computer equipment is lover than 40%.

**Prob. 44**AP Stats 2016 The College Board reported that 60.3% of all students who took the 2016 AP Statistics exam earned scores of 3 or higher. One teacher wondered if the performance of her school was better. She believed that year’s students to be typical of those who will take AP Stats at that school and was pleased when 34 of her 54 students achieved scores of 3 or better.

a) How many standard deviations above the national rate did her students score? Does that seem like a lot? Explain.

b) Can she claim that her school is better? Explain.

In [25]:
def p44():
    n = 54
    k = 34
    p_hat = k / n
    p0 = 0.603
    q0 = 1 - p0
    sd_p_hat = sqrt(p0 * q0 / n)
    z = (p_hat - p0) / sd_p_hat
    p_val = 1 * norm.cdf(z)

    print (f"p_hat: {p_hat:.3f}")
    print (f"sd_p_hat: {sd_p_hat:.3f}")
    print (f"z: {z:.3f}")
    print (f"p_val: {z:.3f}")
p44()

p_hat: 0.630
sd_p_hat: 0.067
z: 0.400
p_val: 0.400


**Answers to Prob. 44**

(a) Let p = propotion of students achieves scores of 3 or higher. $H_0:p$ = 0.603 vs. $H_A:p$ =/ 0.603
It is 0.4 standard deviation of her students score above the mean.  The P-value of 0.4 says that her students were still at 0.603, then the observed proportion as different as 0.63 of her students achieved scores of 3 or higher at a random about 40 times in 100.  

(b) With a P-value this large, I retain the null hypothesis.  This is strong evidence that the propotion of students is equal to the baseline for the all students who took the 2016 AP Statistics exam earned scores of 3 or higher.
